draft-ietf-taps-interface.md

```diff
@@ -1740,9 +1767,29 @@ The ReceiveError event passes an optional associated MessageContext. This may
 indicate that a Message that was being partially received previously, but had not
 completed, encountered an error and will not be completed.
 
-## Message Receive Context {#receive-context}
+## Message Contexts {#msg-ctx}
+
+A MessageContext object contains properties, see {{message-props}}, and other
+metadata for Messages. Therefore, a MessageContext object can be passed to the
+Send Action and is retuned by each Send action as well as Send and Receive
+related events.
+
+The application can set, see {{message-props}} and query Message Properties using the Message Context.
+
+~~~
+PropertyValue := MessageContext.Get(property)
+~~~
+
+The application can also query information about the local and remote endpoint.
+
+~~~
+RemoteEndpoint := MessageContext.GetRemoteEndpoint()
+LocalEndpoint := MessageContext.GetLocalEndpoint()
+~~~
+
+Message Contexts can also be used to send messages that are flagged as reply to other messages, see {{send-replies}} for details.
```

Comment:
I think (not 100% sure) that there must be an "a" before "reply" here. Anyway, if you add the "a", I'm quite sure it's correct.